assignment7/src/components/DeveloperCards/FollowUp.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+
+const FollowUp = ({ url, closeModal, header }) => {
+  console.log(url, closeModal);
+  const [follows, setFollows] = useState([]);
+  useEffect(() => {
+    getFollows();
+  }, []);
+  async function getFollows() {
+    const resp = await fetch(`${url}`);
+    const data = await resp.json();
+    setFollows(data);
+  }
+  const fellows = follows?.map((user) => (
+    <div key={user.id} className="fellow">
+      <div className="fellow_details">
+        <img src={user.avatar_url} alt="" />
+        <p className="fellow_username">{user.login}</p>
+      </div>
+      <a target="_blank" href={`https://github.com/${user.login}/`}>
+        follow
+      </a>
+    </div>
+  ));
```

Comment:
```suggestion
const FollowUp = ({ url, closeModal, header }) => {
  const [follows, setFollows] = useState([]);
  
  useEffect(() => {
    getFollows();
  }, []);
  
  async function getFollows() {
    const resp = await fetch(`${url}`);
    const data = await resp.json();
    setFollows(data);
  }
  
  const fellows = follows?.map((user) => (
    <div key={user.id} className="fellow">
      <div className="fellow_details">
        <img src={user.avatar_url} alt="" />
        <p className="fellow_username">{user.login}</p>
      </div>
      <a target="_blank" href={`https://github.com/${user.login}/`}>
        follow
      </a>
    </div>
  ));
```
Add spacing between your declarations, functions and return statement. Remove console logs